api/resources_portal/middleware/oauth.py

```diff
@@ -48,7 +49,11 @@ def __call__(self, request):
             }
 
             # get user orcid info
-            URL = "https://sandbox.orcid.org/oauth/token"
+            if OAUTH_ENVIRONMENT == "SANDBOX":
+                URL = "https://sandbox.orcid.org/oauth/token"
+            else:
+                URL = "https://orcid.org/oauth/token"
```

Comment:
Why do this rather than just make the URL the env var?

Reply:
Oh dear, this was from an older version- looks like it didn't save correctly. Fixing.